assets/properties-panel.css

```diff
@@ -262,7 +261,12 @@
 }
 
 .bio-properties-panel-description {
-  margin-top: 7px;
+  display: block;
+  margin: 4px 0 8px;
+  color: #999;
```

Comment:
Can we try to use [semantic color variables here](https://github.com/bpmn-io/properties-panel/blob/44d12e9c085030d935e628063554627596c71c90/assets/properties-panel.css#L14)?

So something like

```css
.bio-properties-panel {
  --color-999999: #999999;

  --color-description: var(--color-999999);
}

.bio-properties-panel-description {
  color: var(--color-description);
}
```

Reply:
:+1: good point